src/components/dictionaryConcepts/components/MappingModal.jsx

```diff
@@ -101,8 +91,6 @@ export class MappingModal extends Component {
             )}
           </ModalBody>
           <ModalFooter>
-            <Button color="primary" id="mappingSubmit" onClick={to_concept_name !== '' ? this.EditMapping : null}>Save</Button>
```

Comment:
Is this related to this ticket?

Reply:
Yes. The Save button has to be removed because the Edit functionality is being removed too